File: counting.swift
import Foundation

let startTime = Date().timeIntervalSince1970
var count = 0
for _ in 1...1_000_000_000 {
    count += 1
}
let endTime = Date().timeIntervalSince1970
print("Count: \(count)")
print("Execution time: \(endTime - startTime) seconds")

File: counting.ts
let count: number = 0;
let startTime: number = new Date().getTime();
for (let i = 1; i <= 1000000000; i++) {
  count++;
}
let endTime: number = new Date().getTime();
console.log(`Count: ${count}`);
console.log(`Execution time: ${(endTime - startTime) / 1000} seconds`);